docs/unmanaged_configuration.md

```diff
@@ -25,3 +25,50 @@ oc set env dc/fluentd MERGE_JSON_LOG=true
 * Possible log loss due to Elasticsearch rejecting documents due to inconsistent type mappings
 * Potential buffer storage leak caused by rejected message cycling
 * Overwrite of data for field with same names
+
+## ElasticSearch
+### openshift_logging_es_allow_external
```

Comment:
We should make no references to ansible style variables here.  The wording should be along the lines of "Exposing elasticsearch service with a route".   The goal is to describe how to enable features from 3.11 but avoid mentioning ansible since it isn't what we are using in 4.x.

Reply:
Thanks.  Updated.